stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldController.kt

```diff
@@ -150,16 +147,4 @@ class SimpleTextFieldController constructor(
     override fun onFocusChange(newHasFocus: Boolean) {
         _hasFocus.value = newHasFocus
     }
-
-    companion object {
-        fun createEmailSectionController(initialValue: String?) = SimpleTextFieldController(
-            EmailConfig(),
-            initialValue = initialValue
-        )
-
-        fun createNameSectionController(initialValue: String?) = SimpleTextFieldController(
-            NameConfig(),
-            initialValue = initialValue
-        )
-    }
```

Comment:
controller implementations were coupled to TextFieldController. now each `Config` can build its own controller.